downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -76,6 +76,10 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### Ballerina Update Tool
+
+- Improved ballerina distribution update commands (`bal dist update`, `bal dist pull latest`).
```

Comment:
@udda1996 Shall we elaborate more by mentioning `dist update` will do a patch version bump and we need to use `latest` for minor version bump